java/code/webapp/errors/403.jsp

```diff
@@ -0,0 +1,30 @@
+<%@ taglib uri="http://struts.apache.org/tags-bean" prefix="bean" %>
+<%@ taglib uri="http://www.opensymphony.com/sitemesh/page" prefix="page" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
+<%@ taglib uri="http://rhn.redhat.com/rhn" prefix="rhn"%>
+<%@ page contentType="text/html; charset=UTF-8" %>
+
+<page:applyDecorator name="layout_c">
```

Comment:
This is a new page for processing the 403 return code. Did you test it when calling the download endpoint, to make sure this page is not used?